config/routes.rb

```diff
@@ -2,6 +2,8 @@
   devise_for :users
```

Comment:
i think you didn't pull my PR with omniauth because the omniauth callbacks line is not here

Reply:
yeah I haven't pull yours yet because I was gonna pull it when mine is merged and do it altogether